### ISSUE TITLE
Can you provide the loss curve during training?

### ISSUE THREAD

**Issue by Sparrowwwwho** [multimediaFor/EITLNet#9]
作者您好，目前在试图复现文章的结果，但是在训练时val loss中间有很异常的飙升，请问作者可以提供一下训练过程的损失曲线图吗？

**Comment by multimediaFor** [comment]
Okay, this is a version I retained during the training process (it may not correspond to the publicly available weights, but the loss convergence trend is essentially the same.
![WechatIMG5974](https://github.com/user-attachments/assets/5cca9b8d-92b0-4ec8-b7ca-e5fadd88a966)

**Comment by Sparrowwwwho** [comment]
感谢回答，但是我多次使用您的默认代码和60971数据集进行训练，train_loss下降趋势比作者您的曲线要更慢，val_loss第一轮第二轮永远是0.08和0.07左右，后续直到40轮一直在0.09上下浮动，根据您代码的best_epoch_weights.pth保存逻辑，它只在第一轮和第二轮进行了保存，后续再也没有保存了。
请问作者您在24.04.26给出的权重文件中，学习率是文章中的5e-3 to 5e-4还是代码中的1e-4 to  1e-6?
数据集方面,train.txt和val.txt训练还是先用train_del_ca2.txt和val_del_ca2.txt训练然后用casia2微调呢？

**Comment by Qishiyii** [comment]
你好，最后的问题解决了吗？

**Comment by Qishiyii** [comment]
你好，最好您的问题解决了吗？如果方便可不可以交流一下

**Comment by Sparrowwwwho** [comment]
完整进行了两次训练，val loss比作者的值要高，波动也大一点，但四十轮开始波动幅度就减少了

**Comment by Qishiyii** [comment]
你好，我也是这个研究方向，但是我最近才开始，可不可以向你请教一些问题，如果可以，加我联系V Q13935721044，如果不方便，也十分感谢你的解答

**Comment by Sparrowwwwho** [comment]
亲亲这边建议快逃呢，我给师弟师妹的第一个建议就是不要搞这个方向

**Comment by Qishiyii** [comment]
为什么呢？师兄

**Comment by Qishiyii** [comment]
为什么呢师兄，我这边跑不了，再过几天开题了，太难搞了

**Comment by Qishiyii** [comment]
师兄有没有推荐可以参考的论文，我的师兄师姐他们都不是这个方向的，我找了很多论文，但是都是没有代码，前一周复现了一篇，最好发现源码错误太多了，就放弃了

**Comment by Sparrowwwwho** [comment]
我也一头雾水= =

**Comment by Qishiyii** [comment]
好的，麻烦师兄了，我这两天在https://github.com/greatzh/Papers找，很多论文感觉不错，就是都不附带代码，让人生气，自己又没有手搓代码的能力，哎还是太菜了，师兄可以找一找

**Comment by whbwstc** [comment]
Hello,  how long is each round of model training? I use code to train directly on A800, and it takes about 15 hours for a epoch. How did this happen? I look forward to your answer very much, which is very important to me.

> 感谢回答，但是我多次使用您的默认代码和60971数据集进行训练，train_loss下降趋势比作者您的曲线要更慢，val_loss第一轮第二轮永远是0.08和0.07左右，后续直到40轮一直在0.09上下浮动，根据您代码的best_epoch_weights.pth保存逻辑，它只在第一轮和第二轮进行了保存，后续再也没有保存了。 请问作者您在24.04.26给出的权重文件中，学习率是文章中的5e-3 to 5e-4还是代码中的1e-4 to 1e-6? 数据集方面,train.txt和val.txt训练还是先用train_del_ca2.txt和val_del_ca2.txt训练然后用casia2微调呢？

**Comment by Sparrowwwwho** [comment]
> Hello, how long is each round of model training? I use code to train directly on A800, and it takes about 15 hours for a epoch. How did this happen? I look forward to your answer very much, which is very important to me.
> 
> > 感谢回答，但是我多次使用您的默认代码和60971数据集进行训练，train_loss下降趋势比作者您的曲线要更慢，val_loss第一轮第二轮永远是0.08和0.07左右，后续直到40轮一直在0.09上下浮动，根据您代码的best_epoch_weights.pth保存逻辑，它只在第一轮和第二轮进行了保存，后续再也没有保存了。 请问作者您在24.04.26给出的权重文件中，学习率是文章中的5e-3 to 5e-4还是代码中的1e-4 to 1e-6? 数据集方面,train.txt和val.txt训练还是先用train_del_ca2.txt和val_del_ca2.txt训练然后用casia2微调呢？

I dont remember too much but definitely less than 2hours for an epoch.(RTX4090)
I think you should check your dataset, cropsize etc.

**Comment by whbwstc** [comment]
> > 您好，请问模型每轮训练要多长时间？我直接在A800上用代码训练，一个epoch大概需要15个小时。请问这是怎么回事？非常期待您的解答，这对我很重要。
> > > 感谢回答，但是我多次使用您的默认代码和训练60971数据集进行，train_loss下降趋势比作者您的曲线要更慢，val_loss第一轮第二轮永远是0.08和0.07左右，后续直到40轮一直在0.09上下浮动，根据您代码的best_epoch_weights.pth保存逻辑，它只在第一轮和第二轮进行了保存，后续再也没有保存了。请问作者您在24.04.26给出的权重文件中，学习率是文章中的5e-3到5e-4还是代码中的1e-4到1e-6？ 数据集方面，train.txt和val.txt训练还是先用train_del_ca2.txt和val_del_ca2.txt训练然后用casia2参数呢？
> 
> 我记不太清楚，但一个纪元肯定少于 2 小时。（RTX4090） 我认为你应该检查你的数据集、裁剪尺寸等。

Thank you very much for your reply！
